packages/api-client/src/giphy/GiphyAPI.ts

```diff
@@ -30,52 +31,85 @@ export class GiphyAPI {
     PROXY: '/proxy',
     RANDOM: 'random',
     SEARCH: 'search',
+    TRENDING: 'trending',
   };
 
   /**
    * Get a Giphy image by its ID.
    * @param ids one or multiple image ID(s)
+   * @see https://developers.giphy.com/docs/api/endpoint#get-gif-by-id
    */
-  public async getGiphyById(ids: string | string[]): Promise<GiphyResult> {
-    const allIds = Array<string>().concat(ids);
+  public async getGiphyById(options: GiphyIdOptions): Promise<GiphyMultipleResult>;
+  public async getGiphyById(ids: string | string[]): Promise<GiphyMultipleResult>;
+  public async getGiphyById(optionsOrIds: string | string[] | GiphyIdOptions): Promise<GiphyMultipleResult> {
     const config: AxiosRequestConfig = {
       method: 'get',
-      url: `${GiphyAPI.URL.PROXY}/${GiphyAPI.URL.GIPHY}/${allIds.join(',')}`,
```

Comment:
This endpoint doesn't exist anymore, see https://developers.giphy.com/docs/api/endpoint#get-gif-by-id.